Patch status: limbo, partially needs upstreaming

Only manifests as an issue with gcc-13+musl which does not transitively
include cstdint / stdint.h for the int32/64_t usage.

Partially fixed by [1] (>=chromium-110.0.5434.0) given it removes the
int64_t usage altogether in seed_response.h.

See also [2] which fixed this for the 87 branch in Qt5.

[1] https://crrev.com/01e21a077a0354d85e8359195613c2781f67a3a2
[2] https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/489133
--- a/src/3rdparty/chromium/components/variations/seed_response.h
+++ b/src/3rdparty/chromium/components/variations/seed_response.h
@@ -7,2 +7,3 @@
 
+#include <cstdint>
 #include <string>
--- a/src/3rdparty/chromium/gpu/config/gpu_driver_bug_workarounds.h
+++ b/src/3rdparty/chromium/gpu/config/gpu_driver_bug_workarounds.h
@@ -7,2 +7,3 @@
 
+#include <cstdint>
 #include <vector>
--- a/src/3rdparty/chromium/third_party/webrtc/rtc_base/system/file_wrapper.h
+++ b/src/3rdparty/chromium/third_party/webrtc/rtc_base/system/file_wrapper.h
@@ -16,2 +16,3 @@
 
+#include <cstdint>
 #include <string>
